### PR TITLE
Fixes bug for uninitialized constant

### PR DESCRIPTION
**Pull request by joshuatobin**
I'm getting an uninitialized constant in staging, but not locally. Opening this up as a discussion starting point. 

```
 Started POST "/api/v1/chat_ops/respond" for ...
Processing by Api::V1::ChatOpsController#respond as JSON
...
Completed 500 Internal Server Error in 34ms (ActiveRecord: 10.2ms)
NameError (uninitialized constant Api::V1::ChatOpsController::ChatOps):
app/controllers/api/v1/chat_ops_controller.rb:11:in `respond'
```

CC: @lexelby @reidmix

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -10,5 +10,6 @@ module Retrodot
   class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
     config.autoload_paths << Rails.root.join('lib', 'chat_ops')
+    config.eager_load_paths += %W( #{config.root}/lib/chat_ops )
```
Comment (reidmix) [inline config/application.rb]: you don't need `auto_load path` anymore.

File: config/application.rb [hunk 2]
```diff
@@ -8,7 +8,11 @@
 
 module Retrodot
   class Application < Rails::Application
-    config.autoload_paths << Rails.root.join('lib')
-    config.autoload_paths << Rails.root.join('lib', 'chat_ops')
+    config.autoload_paths   += %W( #{config.root}/lib )
+    config.autoload_paths   += %W( #{config.root}/lib/chat_ops )
```
Comment (reidmix) [inline config/application.rb]: Can be just one line:
```ruby
config.autoload_paths   += %W( #{config.root}/lib  #{config.root}/lib/chat_ops  )
```

**Comment by joshuatobin** [comment]
@reidmix should be all set

**Comment by reidmix** [comment]
LGTM 👍

**Pull request merged**